openshift-ci/Dockerfile.tools

```diff
@@ -1,22 +1,30 @@
 FROM fedora:31
+
 ENV GOPATH /go
 ENV GOBIN /go/bin
 ENV GOCACHE /go/.cache
-ENV PATH=$PATH:$GOPATH/bin
-ENV GOVERSION golang-1.13.5-1.fc31
+ENV GOVERSION 1.13.5
+ENV PATH=$PATH:/root/.gimme/versions/go"$GOVERSION".linux.amd64/bin:$GOBIN
 ARG GO_PACKAGE_PATH=github.com/openshift-kni/performance-addon-operators
 
 # rpms required for building and running test suites
 RUN dnf -y install \
-    $GOVERSION \
+    git \
```

Comment:
can you add gcc please? That should be it then.

Reply:
done